src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(({ sex }) => sex === 'm');
+
+  (arguments.length === 2)
+    && (men = men.filter(({ died }) => Math.ceil(died / 100) === century));
```

Comment:
do this inside first filter, also check for century variable, not for arguments.length